Sources/FeaturevisorSDK/Instance+Evaluation.swift

```diff
@@ -652,3 +651,19 @@ extension FeaturevisorInstance {
         return value
     }
 }
+
+extension FeaturevisorInstance {
```

Comment:
Please remove this extension as redundant. You already have `toDictionary` as an extension to `Evaluation`.